Metaverse/Metaverse.uproject

```diff
@@ -19,9 +22,8 @@
 			]
 		},
 		{
-			"Name": "CronosPlayUnreal",
-			"Enabled": true,
-			"MarketplaceURL": "com.epicgames.launcher://ue/marketplace/product/4488d810074049ac988e755a3d9dc0c4"
```

Comment:
should this wait for a new version of the unreal plugin to be published?

Reply:
We can keep using the git plugin, since it is easier to test on CI. And add a new branch for marketplace one later.